src/vs/vscode.d.ts

```diff
@@ -3755,6 +3755,42 @@ declare module 'vscode' {
 		provideSignatureHelp(document: TextDocument, position: Position, token: CancellationToken, context: SignatureHelpContext): ProviderResult<SignatureHelp>;
 	}
 
+	/**
+	 * Argument label information of signature.
+	 */
+	export class SignautreArgumentsLabel {
+		/**
+		 * The name of the parameter.
+		 */
+		name: string;
```

Comment:
note: in roslyn we support classified text in our 'hints', not just plain text.

Reply:
We also support the user hovering over these hints to get additional information (for example, what quick-info would show for teh entity in the hint).

Reply:
for reference: https://github.com/dotnet/roslyn/blob/18d8e7c9a135e0b042a079823d9ef95c63b84b33/src/Features/Core/Portable/InlineHints/InlineHint.cs#L16-L18